marklogic-data-hub/src/main/java/com/marklogic/hub/cli/client/CommandLineFlowInputs.java

```diff
@@ -111,10 +114,20 @@
                 runFlowString.append("\n\t\tInput File Path: " + inputFilePath);
                 fileLocations.put("inputFilePath", inputFilePath);
             }
+
+            if (outputURIPrefix != null) {
+                if(outputURIReplacement != null){
```

Comment:
Won't WriteStepRunner throw this error? I think it's fine to just handle it in one place.